src/components/IconButton.stories.tsx

```diff
@@ -121,8 +130,16 @@ export function IconButtonLink() {
 /** Hover styled version of the IconButton */
 function HoveredIconButton(args: IconButtonProps) {
   return (
-    <div css={{ "& button": args.contrast ? iconButtonContrastStylesHover : iconButtonStylesHover }}>
-      <IconButton {...args} />
+    <div
+      css={{
+        "& button": args.contrast
+          ? iconButtonContrastStylesHover
+          : args.circle
+            ? iconButtonCircleStylesHover
+            : iconButtonStylesHover,
+      }}
```

Comment:
question: I'd expect the component to just own these styles, was it obvious when working on this why this story is overriding in this way?

Reply:
I believe this is a storybook hack to force showing hover styles w/o actually triggering a hover with the `play` / etc. 

I.e. I saw quite a bit of these when futzing when emotion-vs-fela selectors, and the button was always "wrap the real component in a div" + "setup div > (whatever element tag): ...consts of hover styles i want to snapshot...".

I don't love it, but it is current practice...

Reply:
Oh right 🤔 - seems like maybe there are more options now that you can do the `play` interactions with elements: https://www.chromatic.com/docs/hoverfocus/